index.html

```diff
@@ -2,30 +2,49 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="/lab-css-instagram-clone/css/main.css">
 </head>
-<body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+<body class="horizontal-stripes">
 
+  <section>
+    <div>
+      <img id="phones" src="/lab-css-instagram-clone/images/phones.png" alt="">
+    </div>
+  <div>
+      <article>
+          <img id="brand" src="/lab-css-instagram-clone/images/brand.png" alt="">
+         <button id="loginbutton"> <img src="/lab-css-instagram-clone/images/home.png" alt=""> Log in</button>
+       </article>
+<div id="text">
+  <h1>Capture and Share </h1>
+  <h1>the World's Moments</h1>
+  <p> 
+  Instagram is a <b> fast, beautiful </b>and <b> fun </b>way to share your life
```

Comment:
The tag <b> is not so good to use, because it generates the style from the html, better to do 
<span class="bold">fun</span>
.bold{
     font-weight: bold;
}